Dockerfile

```diff
@@ -22,3 +22,34 @@ RUN yarn install --frozen-lockfile --check-files --cache-folder .ycache && \
     rm -rf .ycache
 
 COPY . /code/
+
+# -------------------------- Builder ---------------------------------------
+FROM builder AS nginx-build
+
+# Dynamic configs. Can be changed with containers. (Placeholder values)
+ENV APP_TITLE=APP_TITLE_PLACEHOLDER
+ENV APP_ENVIRONMENT=APP_ENVIRONMENT_PLACEHOLDER
+ENV APP_MAPBOX_ACCESS_TOKEN=APP_MAPBOX_ACCESS_TOKEN_PLACEHOLDER
+ENV APP_GOOGLE_ANALYTICS_ID=APP_GOOGLE_ANALYTICS_ID_PLACEHOLDER
+ENV APP_GRAPHQL_API_ENDPOINT=https://APP-GRAPHQL-API-ENDPOINT-PLACEHOLDER.COM/
+
+# Yarn build variables (Requires backend pulled)
+ENV APP_GRAPHQL_CODEGEN_ENDPOINT=./backend/schema.graphql
+
+# FIXME: yarn install required for patch...
+
+RUN yarn install && yarn generate && yarn build
```

Comment:
The patch should have already been applied on the first yarn install. This looks weird!